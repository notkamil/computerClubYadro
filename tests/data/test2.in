1
10:00 12:00
5
10:15 1 client1
10:20 2 client1 1