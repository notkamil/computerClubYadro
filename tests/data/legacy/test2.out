10:00
10:15 1 client1
10:20 2 client1 1
12:00 11 client1
12:00
1 10 01:40